src/bin/cargo-miri.rs

```diff
@@ -167,20 +167,22 @@ fn ask(question: &str) {
     io::stdout().flush().unwrap();
     io::stdin().read_line(&mut buf).unwrap();
     match buf.trim().to_lowercase().as_ref() {
-        "" | "y" | "yes" => {}, // proceed
+        "" | "y" | "yes" => (), // proceed
         "n" | "no" => show_error(format!("Aborting as per your request")),
         a => show_error(format!("I do not understand `{}`", a))
     };
 }
 
-/// Perform the setup requires to make `cargo miri` work: Getting a custom-built libstd. Then sets MIRI_SYSROOT.
-/// Skipped if MIRI_SYSROOT is already set, in that case we expect the user has done all this already.
+/// Performs the setup requires to make `cargo miri` work, getting a custom-built libstd,
+/// then sets `MIRI_SYSROOT`.
+/// Skipped if `MIRI_SYSROOT` is already set, in that case we expect the user has done all this
+/// already.
 fn setup(ask_user: bool) {
     if std::env::var("MIRI_SYSROOT").is_ok() {
         return;
     }
 
-    // First, we need xargo
+    // First, we need xargo.
```

Comment:
FTR, I agree with this full stop -- there's a proper grammatical sentence before it.